inference-engine/tests/functional/plugin/shared/include/single_layer_tests/tensor_iterator.hpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <tuple>
+#include <string>
+#include <vector>
+#include <memory>
+#include <ngraph/op/util/attr_types.hpp>
+#include "functional_test_utils/layer_test_utils.hpp"
+#include "ngraph_functions/builders.hpp"
+#include "ngraph_functions/utils/ngraph_helpers.hpp"
+
+namespace LayerTestsDefinitions {
+
+enum class TensorIteratorBody {
```

Comment:
Looks like, we can move this enum to [`ngraph_helpers`](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/ngraph_functions/include/ngraph_functions/utils/ngraph_helpers.hpp) (Can be reused in the future)

Reply:
done